runtime/lua/vim/lsp/log.lua

```diff
@@ -32,6 +32,8 @@ do
 
   vim.fn.mkdir(vim.fn.stdpath('cache'), "p")
   local logfile = assert(io.open(logfilename, "a+"))
+  -- Start message for logging
+  logfix:write(string.format("[ START ] LSP logging initiated: %s\n", os.date(log_date_format)))
```

Comment:
```suggestion
  logfile:write(string.format("[ START ] LSP logging initiated: %s\n", os.date(log_date_format)))
```

Typo

Might also be worth sticking to the format of the regular log messages and move the date further to the start?


```suggestion
  logfile:write(string.format("[ START ] %s ] LSP logging initiated\n", os.date(log_date_format)))
```